Nightly GitHub Actions Build Fail on Fri, March 10th 2023

See run for more details:
https://github.com/TileDB-Inc/TileDB/actions/runs/4380912982

File missing from 'dev'?

```

8s
Run cmake -B build -DTILEDB_WERROR=ON -DTILEDB_SERIALIZATION=ON -DCMAKE_BUILD_TYPE=Release
-- Building for: Visual Studio 17 2022
CMake Error at C:/Program Files/CMake/share/cmake-3.25/Modules/CMakeDetermineSystem.cmake:130 (message):
  Could not find toolchain file:
  D:/a/TileDB/TileDB/.github/misc/github_windows_toolchain.cmake
Call Stack (most recent call first):
  CMakeLists.txt:140 (project)
```

--- a/.github/workflows/nightly-test.yml
+++ b/.github/workflows/nightly-test.yml
@@ -57,7 +57,6 @@ jobs:
         if: contains(matrix.os, 'windows')
         env:
           CXXFLAGS: ${{ matrix.cxxflags }}
-          CMAKE_TOOLCHAIN_FILE: ${{ github.workspace }}/.github/misc/github_windows_toolchain.cmake
         run: |
           cmake -B build -DTILEDB_WERROR=ON -DTILEDB_SERIALIZATION=ON -DCMAKE_BUILD_TYPE=Release
 

